tests/boardfarm_plugins/boardfarm_prplmesh/tests/ap_config_renew.py

```diff
@@ -0,0 +1,66 @@
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+import time
+
+from .prplmesh_base_test import PrplMeshBaseTest
+from capi import tlv
+
+
+class ApConfigRenew(PrplMeshBaseTest):
+    """Check initial configuration on device."""
+
+    def runTest(self):
+        # Locate agents and controller
+        for dev in self.dev:
+            if dev.controller_entity:
+                controller = dev.controller_entity
+            if dev.agent_entity:
+                agent = dev.agent_entity
+
+        dev.wired_sniffer.start(self.__class__.__name__ + "-" + dev.name)
+        mac_repeater1_upper = agent.mac.upper()
```

Comment:
Please preserve the comments:
```suggestion
        # Regression test: MAC address should be case insensitive
        mac_repeater1_upper = agent.mac.upper()
```

Reply:
Thanks for noticing.